desktop/apps/artwork/client/index.coffee

```diff
@@ -72,9 +73,20 @@ module.exports =
           require('../../../components/current_auctions/index.jsx').default unless context.is_closed
           require '../components/artist_artworks/index.coffee' if context.is_closed
           require '../components/related_artworks/index.coffee' if context.is_closed
-          require '../components/related_artists/index.coffee'
+          require '../components/related_artists/index.coffee',
         ]
 
+      # Make some Auctions-only layout tweaks. See https://github.com/artsy/auctions/issues/360#event-1017559326
+      # for more info.
+      postInit: ->
+        $currentAuctions = $ '.artwork-current-auctions'
+        $relatedRail = $ '.artwork-artist-related-rail'
+        paddingTop = if $currentAuctions.length then '42px' else '31px'
+
+        $relatedRail
+          .css('border-top', 0)
+          .css('padding-top', paddingTop)
```

Comment:
Another thought that occurred to me:

In a BEM design, these modules would be in and of themselves, and also elements of some sort of fold or footer block. Let's call it `.artist-fold`, and then the elements would be `.artist-fold__auction-artworks`, `artist-fold__current-auctions` and `.artist-fold__related-artists`. If we were factored this way more explicitly, I'd say the real thing to do would be to make sure that the `artist-fold` block-level styling of these treated these elements as opaque rectangles and provided all the spacing control. Then the individual subblocks like `auction-artworks` would style only the insides of their own bounding boxes.